stuffer/s2n_stuffer.c

```diff
@@ -35,6 +35,12 @@ bool s2n_stuffer_is_valid(const struct s2n_stuffer* stuffer)
            stuffer->read_cursor <= stuffer->write_cursor;
 }
 
+extern bool s2n_stuffer_reservation_is_valid(const struct s2n_stuffer_reservation* reservation)
```

Comment:
The `extern` was a copy & paste "typo", I removed it.

https://github.com/awslabs/s2n/pull/1990 is tracking your request to switch to a `_validate` style. This will break our current proofs, so I'll implement that on a separate PR.